README.md

```diff
@@ -51,13 +51,24 @@ rather than IP numbers in a Hyper-V environment. On CentOS systems,
 this requires the bind-utils package (updatedns requires the 'dig'
 program). The bind-utils package isn't otherwise necessary.
 
+- Install minimum version of Ruby (the system must have 1.9 or higher, but 2.2.6 is recommended)
```

Comment:
Also, note the commands for the systems (sudo yum install or sudo apt-get) may be installing the GCC compiler, which ruby will need). I think this entire section should be BELOW the system setup instructions (at the end). That way, you're guaranteed to have GCC.